### ISSUE TITLE
Dynamically set the expandDrawable and collapseDrawable

### ISSUE THREAD

**Issue by charlenelu** [Manabu-GT/ExpandableTextView#48]
Can provide two public methods to dynamically set the  expandDrawable and collapseDrawable ?

**Comment by zhuyifan2013** [comment]
Agree.  it is not wise  for " TextView and ImageButton must be set to "@id/expandable_text" and "@id/expand_collapse" "
